web/src/main/java/com/futurewei/alcor/web/entity/NetworkVlanRangeRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.futurewei.alcor.web.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class NetworkVlanRangeRequest {
```

Comment:
Split it to NetworkValanRangeRequest and Response

Reply:
Now I make it as NetworkRangeRequest. All network type use the same range request class

Reply:
Cool thanks for making this change.